features/beacon.yml

```
@@ -0,0 +1,5 @@
+name: Beacon
```

The spec refers to these as "beacons." I think that fits in well with other noun features we have. The ID would have to change too though:

```suggestion
name: Beacons
```